lib/src/dialogs/macos_alert_dialog.dart

```diff
@@ -144,6 +183,13 @@ class MacosAlertDialog extends StatelessWidget {
               ],
             ],
             const SizedBox(height: 16),
+            if (suppress != null) ...[
```

Comment:
I don't think the spread operator is needed because only one widget is being added to the list.